### ISSUE TITLE
Screen reader classes

### ISSUE THREAD

**Issue by haroldangenent** [trendwerk/trendpress#538]
We might need to implement these.

https://codex.wordpress.org/CSS#WordPress_Generated_Classes

**Comment by luukdv** [comment]
I don't know.. maybe for specific websites where accessibility is very important, but not the framework in general?

**Comment by haroldangenent** [comment]
Accessibility is always very important.

> These people need access to web pages, and as a web developer, you need to know about accessibility.
https://codex.wordpress.org/Accessibility

However, this is _not_ a quick fix. Almost all HTML output should be altered to include screen reader text (https://make.wordpress.org/accessibility/tag/screen-reader/). Any ideas on what to do?
